src/pages/MainPage/CitiesSection.jsx

```diff
@@ -5,17 +5,48 @@ import CityCard from '../../components/common/CityCard/CityCard';
 const CitiesSection = () => {
   return (
     <section className="cities">
-      <div className="cities-wrapper">
-        <CityCard />
-        <CityCard />
-        <CityCard />
-        <CityCard />
-        <CityCard />
-        <CityCard />
-        <CityCard />
-        <CityCard />
-        <CityCard />
-        <CityCard />
+      <div className="cities-container">
+        <h2 className="cities-title">Popular Cities</h2>
+        <div className="cities-info">
+          <h3 className="cities-subtitle">
+            Explore city guides for your next adventure
+          </h3>
+          <a href="#" className="cities-link-viewAll">
+            View all
+          </a>
+        </div>
+        <ul className="cities-cards-wrapper">
+          <li>
+            <CityCard />
```

Comment:
Instead of being driven by JSX it can be driven by data

you can create a mocked array of cities and then use it in jsx
```js
const cities = [{ id: '123', name: 'Paris', countryName: 'France' }]
...

{cities.map((city) => {
        return <CityCard key={city.id} cityName={city.name} countryName={city.countryName} />;
      })}

```

Reply:
anyway you will be doing this way later when you add an integration with BE

Reply:
Add props to the `CityCard` component so it can accept city name, country name, image

Reply:
Yep, u r rigth! Thnx!